client/src/components/JobInformation/JobInformation.vue

```diff
@@ -67,6 +74,9 @@ import DecodedId from "../DecodedId.vue";
 import CodeRow from "./CodeRow.vue";
 import UtcDate from "components/UtcDate";
 import CopyToClipboard from "components/CopyToClipboard";
+import JOB_STATES_MODEL from "mvc/history/job-states-model";
+import intervalToDuration from "date-fns/intervalToDuration";
+import formatDuration from "date-fns/formatDuration";
```

Comment:
```suggestion
import { formatDuration, intervalToDuration  } from "date-fns";
```

Reply:
(minor, but this import syntax is preferable -- webpack's tree shaking should work cleanly and keep inclusion minimal)